lib/python/util.py

```diff
@@ -34,3 +34,21 @@ def list_files_recursively(dir, pattern):
             matches.append(os.path.join(root, filename))
 
     return matches
+
+def get_git_revision_script_url(file_path):
+    import os
+    import subprocess
```

Comment:
@mhidas agree with you, however the reason I did this is written in the above lines https://github.com/aodn/data-services/blob/00c65c91cac4b748cb67b9b0ac1264b27c56be3d/lib/python/util.py#L5

Since it will hold various functions, In case some rewriting would be required, I decided to make the functions a bit more independent. what do you think ?